README.md

```diff
@@ -163,6 +163,82 @@ import 'octane-components/dist/components/PaymentSubmission/PaymentSubmission.cs
 
 ![Screenshot of the PlanPicker component](./docs/payment-submission.png)
 
+## Actions
+
+In addition to components, `octane-components` provides access to "actions", or asynchronous interactions with our API. While these complement our React components nicely, they can be used in any client-side app. They all return Promises and work well with `await`, when available.
+
+### `getActiveSubscription(token): Promise<PricePlan | null>`
+
+`hasPaymentInfo` is a simple action that checks for a customer's active subscription. If it's there, it resolves to the PricePlan details for that subscription. If the customer has no subscription, it resolves to `null`.
```

Comment:
tiny nit: omit "simple" here. Generally a good thing to avoid in docs. Also in this case it's clear that this is an action so you could omit that as well, just to limit any fluff that we can.

Reference: https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher

Reply:
Ah yeah, good call on this (nice article, too)